client/settings/phone-input/index.tsx

```diff
@@ -0,0 +1,127 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect, useState, useRef } from 'react';
+import { __ } from '@wordpress/i18n';
+import intlTelInput from 'intl-tel-input';
+import './style.scss';
+
+/**
+ * Internal dependencies
+ */
+// @ts-ignore: Cannot find module
+// eslint-disable-next-line import/no-unresolved
+import utils from 'iti/utils';
+
+interface PhoneNumberInputProps {
+	value: string;
+	onValidationChange: ( isValid: boolean ) => void;
+	onValueChange: ( value: string ) => void;
+	inputProps: {
+		label: string;
+		ariaLabel: string;
+		name: string;
+	};
+}
+
+const PhoneNumberInput = ( {
+	onValueChange,
+	value,
+	onValidationChange = ( validation ) => validation,
```

Comment:
This technically could be an empty function, but TypeScript keeps complaining about an empty function